ansible/roles/letsencrypt/tasks/letsencrypt_renew.yml

```diff
@@ -4,5 +4,5 @@
     name: "Renew ooni-backend letsencrypt certificates"
     user: "root"
     cron_file: "letsencrypt_renew_certs-{{ app_env }}"
-    special_time: "monthly"
+    special_time: "weekly"
```

Comment:
After how much time since a let's encrypt cert is issues can you renew it? I suspect this being monthly was the reason why it was not working as expected in the past.
Maybe we can even go as far as doing it daily.

Also, what happens when this role is run?

Is the oonibackend taken offline while it is running?